packager/util/util_test.go

```diff
@@ -22,6 +23,26 @@ func TestCertName(t *testing.T) {
 	assert.Equal(t, "Qk83Jo8qB8cEtxfb_7eit0SWVt0pdj5e7oDCqEgf77o", util.CertName(pkgt.B3Certs[0]))
 }
 
+func TestGetDurationToExpiry(t *testing.T) {
+	// Time before the cert validity.
+	beforeCert := time.Date(2019, time.May, 8, 0, 0, 0, 0, time.UTC)
+	// Time after the cert validity.
+	afterCert := time.Date(2019, time.August, 8, 0, 0, 0, 0, time.UTC)
+	// Time 2 days before cert validity expiration.
+	twoDaysBeforeExpiry := time.Date(2019, time.August, 5, 5, 43, 32, 0, time.UTC)
+	// Time 0 days, 1 hour before cert validity expiration.
+	zeroDaysBeforeExpiry := time.Date(2019, time.August, 7, 4, 43, 32, 0, time.UTC)
+
+	assert.Equal(t, time.Duration(-1), util.GetDurationToExpiry(pkgt.B3Certs[0], beforeCert))
+	assert.Equal(t, time.Duration(-1), util.GetDurationToExpiry(pkgt.B3Certs[0], afterCert))
+	assert.Equal(t, time.Duration(2), util.GetDurationToExpiry(
```

Comment:
I'd suggest changing it to be `2 * 24 * time.Hour` so there's no loss of fidelity in the test (rounding due to integer division). Also, `time.Duration(2)` reads like "2 nanoseconds" to me.